### PR TITLE
ceph-volume: Nits noticed while studying code

### PR DESCRIPTION
**Pull request by dmick**
Spelling/typos, unused function, small formatting bug

### CODE REVIEW

File: src/ceph-volume/ceph_volume/api/lvm.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ def remove_vg(vg_name):
     """
     Removes a volume group.
     """
-    fail_msg = "Unable to remove vg %s".format(vg_name)
+    fail_msg = "Unable to remove vg %s" % vg_name
```
Comment (alfredodeza) [inline src/ceph-volume/ceph_volume/api/lvm.py]: woah, great catch! thanks for fixing these

**Comment by alfredodeza** [comment]
jenkins test ceph-volume tox

File: src/ceph-volume/ceph_volume/api/lvm.py (hunk 1 above)
Comment (dmick) [reply src/ceph-volume/ceph_volume/api/lvm.py]: Heh, thank VS Code

**Comment by dmick** [comment]
Do we not want this in master?

**Comment by alfredodeza** [comment]
@dmick good catch, we were trying to get things moving for 12.2.5 and I missed out on having this merged. Thanks for the ping

**Pull request merged**